cmd/catchpointdump/database.go

```diff
@@ -53,5 +53,8 @@ var databaseCmd = &cobra.Command{
 		if err != nil {
 			reportErrorf("Unable to print account database : %v", err)
 		}
+		if outFileName != "" {
```

Comment:
maybe `defer outFile.close` inside the `if outFileName != "" {` block above?

Reply:
sounds like a good idea.